.travis.yml

```diff
@@ -262,7 +262,7 @@ jobs:
           - if [ -n "${BUILDER_NAME}" ]; then rm -rf /home/${BUILDER_NAME}/* && echo "Cleared /home/${BUILDER_NAME} directory" || echo "Failed to clean /home/${BUILDER_NAME} directory"; fi;
           - if [ -d "${PACKAGES_DIRECTORY}" ]; then rm -rf "${PACKAGES_DIRECTORY}"; fi;
 
-    - name: "Build & Publish DEB package for ubuntu/bionic"
+      name: "Build & Publish DEB package for ubuntu/bionic"
```

Comment:
why this change?

https://github.com/prologic/netdata/blob/8c5cf82cbbb6061a6f7f0c0830d2c8276f212f6b/.travis.yml#L265-L295

Reply:
This was a mistake. `git co -b ...` un existing commits. Fixed.